HW1/P2/P2.py

```diff
@@ -29,3 +30,33 @@ def draw_image(rdd):
     im[I, J] = np.log(C + 1)  # log intensity makes it easier to see levels
     plt.imshow(im, cmap=cm.gray)
     plt.show()
+
+pixel = range(2000)
+pix1D=sc.parallelize(pixel,10)
+pixels = pix1D.cartesian(pix1D)
+mandrdd = pixels.map(lambda i: (i, mandelbrot((i[1]/500.)-2,(i[0]/500.0)-2)))
+draw_image(mandrdd)
+plt.hist(sum_values_for_partitions(mandrdd).collect())
+plt.savefig("P2a_hist.png")
+plt.xlabel("Iteration Count")
+plt.ylabel("Worker Count")
+
+range2000 = range(2000)
+def randomList(a):
+    b = []
+    for i in range(len(a)):
+        element = random.choice(a)
+        a.remove(element)
+        b.append(element)
+    return b
+
+randomlist = randomList(range2000)
```

Comment:
Ah, you know what: I just realized that this file was an initial commit early on, but i ultimately replaced it with P2b found below (where I randomized partitions instead of the pixel arrangement). I guess i never deleted P2.py from my github? Thought i did. Oh well. 


Thanks for the tip though!